dianna/__init__.py

```diff
@@ -69,18 +70,19 @@ def explain_image(model_or_function, input_data, method, labels, **kwargs):
         One heatmap (2D array) per class.
 
     """
-    if method.upper() == 'KERNELSHAP':
+    if method.upper() == "KERNELSHAP":
```

Comment:
Single or double quotes? See first comment

Reply:
See my comment above. Everything is now back to single quote.